README.md

```diff
@@ -0,0 +1,11 @@
+# Training HuggingFace 🤗 Models on an IPU using Paperspace Gradient
+
+Whether you are looking to generate images with Stable Diffusion, derive insights from text, or need to recognize audio samples, the examples in here have you covered. And by using Paperspace to access IPUs you can be up and running in minutes!
+
+See the [README notebook](README_first.ipynb) for more detail on the tasks which are covered.
+
+## Getting Started with our interactive notebooks
+
+This repository contains notebooks which are meant to be run on Paperspace Gradient's IPU Machines:
+
```

Comment:
```suggestion
Run the notebooks in this repository on our configured Paperspace Gradient's IPU Machines:
```
This feels more actionable to me because this readme is small, and the point is to get the user straight to the runtime